python/search_completer/global_search_result_delegate.py

```diff
@@ -0,0 +1,83 @@
+# Copyright (c) 2015 Shotgun Software Inc.
+#
+# CONFIDENTIAL AND PROPRIETARY
+#
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit
+# Source Code License included in this distribution package. See LICENSE.
+# By accessing, using, copying or modifying this work you indicate your
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
+# not expressly granted therein are reserved by Shotgun Software Inc.
+
+import sgtk
+from sgtk.platform.qt import QtCore
+
+from .search_result_delegate import SearchResultDelegate
+
+# import the shotgun_model and view modules from the shotgun utils framework
+shotgun_model = sgtk.platform.import_framework("tk-framework-shotgunutils", "shotgun_model")
+shotgun_globals = sgtk.platform.import_framework("tk-framework-shotgunutils", "shotgun_globals")
+
+
+views = sgtk.platform.current_bundle().import_module("views")
+
+
+class GlobalSearchResultDelegate(SearchResultDelegate):
+    """
+    Delegate which renders search match entries in the global
+    search completer.
+    """
+
+    def _render_result(self, widget, model_index):
+
+        from .global_search_completer import GlobalSearchCompleter
```

Comment:
Most of the code from this class has been moved to SearchResultDelegate. The code itself hasn't changed, expect that it now underlines the search terms in the result.